src/odemis/driver/zeiss.py

```diff
@@ -1116,9 +1116,9 @@ def _move_to_pos(self, pos):
         pos = numpy.append(pos, self.pos[len(pos):])
         self.target_pos = numpy.array(pos)
         self._start_move = time.time()
-        dur_lin = sum(numpy.sqrt((self.pos[[0, 1, 2, 5]] - self.target_pos[[0, 1, 2, 5]]) ** 2)) / self._speed_lin
-        dur_rot = sum(abs(self.pos[[3, 4]] - self.target_pos[[3, 4]])) / self._speed_rot
-        dur = dur_lin + dur_rot
+        dur_lin = max(numpy.sqrt((self.pos[[0, 1, 2, 5]] - self.target_pos[[0, 1, 2, 5]]) ** 2)) / self._speed_lin
```

Comment:
Could you comment on this in the code, why did you switch to max? Does this better represent the real situation for example or were you forced to change this due to increasing the linear move speed?